src/renderer/widgets/CreateWallet/model/__tests__/signatory-model.test.ts

```diff
@@ -0,0 +1,69 @@
+import { allSettled, fork } from 'effector';
+
+import { walletModel } from '@entities/wallet';
+import { initiatorWallet, signatoryWallet, signerWallet } from './mock';
+import { toAddress } from '@shared/lib/utils';
+import { signatoryModel } from '../signatory-model';
+
+describe('widgets/CreateWallet/model/signatory-model', () => {
+  beforeEach(() => {
+    jest.restoreAllMocks();
+  });
+
+  test('should correctly add signatories', async () => {
+    const scope = fork();
+
+    expect(scope.getState(signatoryModel.$signatories).size).toEqual(0);
+
+    await allSettled(signatoryModel.events.signatoriesChanged, {
+      scope,
+      params: { index: 1, name: 'Alice', address: toAddress(signerWallet.accounts[0].accountId) },
+    });
+
+    await allSettled(signatoryModel.events.signatoriesChanged, {
+      scope,
+      params: { index: 0, name: 'test', address: toAddress(signerWallet.accounts[0].accountId) },
+    });
+
+    expect(scope.getState(signatoryModel.$signatories).size).toEqual(2);
+  });
+
+  test('should correctly delete signatories', async () => {
+    const scope = fork();
+
+    expect(scope.getState(signatoryModel.$signatories).size).toEqual(0);
```

Comment:
Do you guys have any idea how this can actually fail? When run alone, this test succeeds.
When run alone, the test succeeds, when the whole file is ran, it fails, and when observing the `$signatories` store, it's indeed not empty! 
How can I make sure that stores are reset between each test?

@Asmadek @johnthecat @sokolova-an

Reply:
I think set it directly should work 

```ts
    const scope = fork({
      values: new Map().set(signatoryModel.$signatories, new Map([])),
    });
```

Reply:
thank you so much, it solved it. I've a hard time understanding why/how it could be non empty though :)